client/webserver/site/src/js/markets.ts

```diff
@@ -2605,18 +2692,22 @@ export default class MarketsPage extends BasePage {
    */
   quantityChanged (finalize: boolean) {
     const page = this.page
-    const order = this.parseOrder()
-    if (order.qty < 0) {
+    const order = this.currentOrder = this.parseOrder()
+    if (order.qty <= 0) {
```

Comment:
I can no longer enter any value < 1, since it requires first entering `0.`, and this now removes the decimal point.

Reply:
Thanks for spotting this, it's fixed.

Reply:
I'm looking into it